internal/container.go

```diff
@@ -86,14 +108,23 @@ func getCheckpointInfo(task Task) (*checkpointInfo, error) {
 
 	info.configDump, _, err = metadata.ReadContainerCheckpointConfigDump(task.OutputDir)
 	if err != nil {
-		return nil, err
+		if strings.Contains(err.Error(), "unexpected end of JSON input") {
+			return nil, fmt.Errorf("config.dump: unexpected end of JSON input")
+		}
+		return nil, fmt.Errorf("config.dump: %w", err)
 	}
 	info.specDump, _, err = metadata.ReadContainerCheckpointSpecDump(task.OutputDir)
 	if err != nil {
-		return nil, err
+		if os.IsNotExist(err) {
+			return nil, fmt.Errorf("spec.dump: no such file or directory")
+		}
+		if strings.Contains(err.Error(), "unexpected end of JSON input") {
+			return nil, fmt.Errorf("spec.dump: unexpected end of JSON input")
+		}
+		return nil, fmt.Errorf("spec.dump: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("spec.dump: %w", err)
```
I don't think we need the special case since we are wrapping with context.